packages/astro/src/i18n/vite-plugin-i18n.ts

```diff
@@ -52,13 +60,16 @@ export default function astroInternationalization({
 						trailingSlash, 
 						format, 
 						site, 
+						isBuild,
 						...i18n,
 						...opts 
 					});
 					
 					export const getRelativeLocaleUrlList = (path = "", opts) => _getLocaleRelativeUrlList({ 
 						base, path, trailingSlash, format, ...i18n, ...opts });
-					export const getAbsoluteLocaleUrlList = (path = "", opts) => _getLocaleAbsoluteUrlList({ base, path, trailingSlash, format, site, ...i18n, ...opts });
+					export const getAbsoluteLocaleUrlList = (path = "", opts) => _getLocaleAbsoluteUrlList({ 
+						base, path, trailingSlash, format, site, isBuild, ...i18n, ...opts 
```

Comment:
looks like `getAbsoluteLocaleUrlList` has not changed but is now being passed `isBuild`, what is this flag used for?

Reply:
I addressed your comment. Very silly from me, good catch. 

> isBuild, what is this flag used for?

From the PR description:

> updated the getAsbsoluteLocaleUrl function, which now accepts domains and a isBuild flag. isBuild is required because we can only use the URLs with domains when building the website, otherwise the dev server is broken